modules/system/rolls/wom-cast-test.js

```diff
@@ -23,6 +30,9 @@ export default class WomCastTest extends CastTest {
     if (this.result.overcast.usage.damage) {
       this.result.overcast.usage.damage.available = this.result.overcast.available >= game.wfrp4e.config.overCastTable.damage[0].cost
     }
+    if (this.result.overcast.usage.extra) {
```

Comment:
should this be usage.extra, or usage.other?

Reply:
Oops, yeah, I wrote extra at one point and then changed it except here